src/models/user.repository.ts

```diff
@@ -0,0 +1,90 @@
+import { User } from "./models";
+import { Item } from "./models";
+
+/**
+ * @desc Create a new user
+ * @param {User} user
+ * @returns {Promise<User | boolean>} Created user or false if user already exists
+ */
+export const createUser = async (user: User): Promise<User | boolean> => {
```

Comment:
Might be worth it to look into an alternate implementation that just returns one specific data type.